.Python

```diff
@@ -0,0 +1 @@
+/System/Library/Frameworks/Python.framework/Versions/2.7/Python
```

Comment:
You don't have to add tests. Just follow the steps in checklist pr.

On Wed, Mar 23, 2016, 07:24 LucaSoldi <notifications@github.com> wrote:

> In .Python
> <https://github.com/balloob/home-assistant/pull/1590#discussion_r57167130>
> :
>
> > @@ -0,0 +1 @@
> > +/System/Library/Frameworks/Python.framework/Versions/2.7/Python
>
> How can I add tests? :)
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/1590/files/65ef836313cfff3a4f14b4fc802b8c457b7c75f6#r57167130>
>